parl/remote/worker.py

```diff
@@ -101,6 +102,23 @@ def _set_cpu_num(self, cpu_num=None):
         else:
             self.cpu_num = multiprocessing.cpu_count()
 
+    def check_version(self):
+        self.request_master_socket.send_multipart(
+            [remote_constants.CHECK_VERSION_TAG])
+        message = self.request_master_socket.recv_multipart()
+        tag = message[0]
+        if tag == remote_constants.NORMAL_TAG:
+            worker_parl_version = parl.__version__
+            worker_python_version = str(sys.version_info.major)
+            assert worker_parl_version == to_str(message[1]) and worker_python_version == to_str(message[2]),\
+                '''Version mismatch: the "master" is of version "parl={}, python={}",however, 
```

Comment:
However, XXX is provided in your environment.

Reply:
modified